jl4-core/src/L4/Lexer.hs

```diff
@@ -269,7 +268,7 @@ directives =
   [ (TStrictEvalDirective, "SEVAL")
   , (TLazyEvalDirective,   "EVAL")
   , (TCheckDirective,      "CHECK")
-  , (TContractDirective,   "CONTRACT")
+  , (TContractDirective,   "TRACE")
```

Comment:
I'd still prefer if we could just make the syntax here first-class, and use `EVAL`.

Also, I'd like to avoid `TRACE` specifically, because perhaps we'll need `TRACE` for evaluation traces.